src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -124,6 +124,11 @@
   {:name schema/Str
    :files [EnvironmentClassesFileEntry]})
 
+(def EnvironmentModulesInfo
+  "Schema for the return payload an environment_classes request."
+  {:name schema/Str
+   :modules schema/Any}) ; TODO: fixme
```

Comment:
Hmm, this is a bit tricker than I thought. At this point, they key `:modules` is still a RubyHash. I'm relying on Ring to convert that map for me when I send it back to the user. Maybe that's a bad idea?

```
Output of module-info-from-jruby->module-info-for-json does not match schema: {:modules [(not (map? a-org.jruby.RubyHash)) (not (map? a-org.jruby.RubyHash))]}
```